server/community.go

```diff
@@ -64,3 +66,62 @@ func (s *Server) postPRWelcomeMessage(ctx context.Context, pr *model.PullRequest
 
 	return nil
 }
+
+func (s *Server) assignGreeter(ctx context.Context, pr *model.PullRequest, repo *Repository) error {
+	// Only assign an greeter for non-member PRs
+	if s.IsOrgMember(pr.Username) {
+		return nil
+	}
+
+	// Is it configured to have a greeting team to handle the PR?
+	if repo.GreetingTeam == "" {
+		return nil
+	}
+
+	// get available members
+	greeterList, _, err := s.GithubClient.Teams.ListTeamMembersBySlug(ctx, s.Config.Org, repo.GreetingTeam, nil)
+	if err != nil {
+		mlog.Error("Error retrieving greeting team members", mlog.Err(err), mlog.String("GreetingTeam", repo.GreetingTeam))
```

Comment:
We can remove this and avoid the double-logging, and add context to the error instead.

Also, we can remove the word "greeting" because that context is already there in the top-level log statement, this is just about the GitHub call failing.